src/test-covariance.cpp

```diff
@@ -126,7 +126,7 @@ context("compound symmetry") {
 context("get_covariance_lower_chol") {
   test_that("get_covariance_lower_chol gives expected unstructured result") {
     vector<double> theta {{log(1.0), log(2.0), 3.0}};
-    matrix<double> result = get_covariance_lower_chol(theta, 2, unstructured_cov);
+    matrix<double> result = get_covariance_lower_chol(theta, 2, "us");
```

Comment:
like here I would have thought we need to make a copy of the string, I remember I had issues before when I tried to pass an int instead of int variable to a const-ref int argument... but probably my C++ is not good enough to understand the difference :-D